src/System.Globalization/src/ApiCompatBaseline.netcore50aot.txt

```diff
@@ -1,2 +1,3 @@
 #Compat issues with assembly System.Globalization (https://github.com/dotnet/corefx/issues/11035) :
-CannotRemoveBaseTypeOrInterface : Type 'System.Globalization.CultureNotFoundException' does not implement interface 'System.Runtime.Serialization.ISerializable' in the implementation but it does in the contract.
\ No newline at end of file
+CannotRemoveBaseTypeOrInterface : Type 'System.Globalization.CultureNotFoundException' does not implement interface 'System.Runtime.Serialization.ISerializable' in the implementation but it does in the contract.
+CannotRemoveBaseTypeOrInterface : Type 'System.Globalization.CultureNotFoundException' does not inherit from base type 'System.SystemException' in the implementation but it does in the contract.
```

Comment:
I don't think this is right, why would netcore50 contract would inherit something from SystemException? netcore50 should have the old contract still.